build.gradle

```diff
@@ -193,6 +193,7 @@ project(':iceberg-bundled-guava') {
 project(':iceberg-api') {
   dependencies {
     implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    implementation "com.fasterxml.jackson.core:jackson-databind"
```

Comment:
Why is an additional Jackson library needed? I think we should be able to add default value serialization just like the other parsers.